e2e/tests/notebook/notebook-create-python.spec.ts

```diff
@@ -52,7 +52,7 @@ describe('Python Kernel Notebook Test', () => {
 
     // Verify kernel name.
     const kernelName = await notebook.getKernelName();
-    expect(kernelName).toBe('Python 3');
+    expect(kernelName).toBe('Python 3 (ipykernel)');
```

Comment:
a const would be nice to have here - for R too

Reply:
Just like other expected values, why a const be useful for once time use local variable?

Reply:
Because it is not single-use.  It is updated twice in this PR.

Reply:
oh you suggest to create a const and shared by both places? Any suggestions about where to put it and how to use that?